<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IsoRecoverDialog</class>
 <widget class="QDialog" name="IsoRecoverDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>475</width>
    <height>121</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ISO Image Recovery</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QLabel" name="mp_label_status">
     <property name="text">
      <string>Processing...</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="mp_progress_bar"/>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="mp_button_box">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>mp_button_box</sender>
   <signal>rejected()</signal>
   <receiver>IsoRecoverDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>237</x>
     <y>100</y>
    </hint>
    <hint type="destinationlabel">
     <x>237</x>
     <y>60</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
